common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -149,6 +149,9 @@ start_function() {
             ;;
         * ) err "unsupported mode: $PRPLMESH_MODE"; usage; exit 1 ;;
     esac
+
+    # Give write premissions
+    sudo chmod -R o+w /tmp/$USER
```

Comment:
```suggestion
    sudo chmod -R o+w /tmp/${SUDO_USER:-${USER}}
```

If I do a local (non-docker) build as user "arnout", everything will go to /tmp/arnout/beerocks. However, I will start prplMesh by running `sudo prplMesh_utils.sh start`so `$USER` will be root instead of arnout.

Reply:
Hmm, I see. I did it in the command line and it worked, so I copied it to the script. I see now that it works a little bit differently when being run from a script.
I'll do as you suggested.

Reply:
> BTW, the 'sudo' at the beginning of the line can be removed? Right?

Indeed, there's no reason for the sudo since the whole script has to be executed as root.